### ISSUE TITLE
Make page titles format consistent across pages

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#1806]
The page title on the main page is "Button Men Online"

Clicking on Forum gives: "Button Men Online - Forum" 
Clicking on Buttons gives: "Button Men Online - Buttons"

Similar for History, Preferences, Who's Online. Clicking into a Game changes the page title as well.

However, the page title does not change on Open games, Profile, Create Game.

Let's make the page title consistent.

Note forum thread http://www.buttonweavers.com/ui/forum.html#!threadId=211 which references how page titles appear in Safari and suggests reversing the order of the titles ("Buttons - Button Men Online").

This also is somewhat related to #1125 (changing page title in monitor mode).

**Comment by dwvanstone** [comment]
From poking around other websites out there (including looking in my browsing history), I see most do reverse the order of the pages once you go down a level:

"Rice - Wikipedia, the free encyclopedia"  
"Apple Watch - Apple"
"Dining Out | Yahoo Answers"
"My Drive - Google Drive"
"Vivica A. Fox | TVGuide.com"
"[subject line] - [email address] - Gmail"
"Sports News - The New York Times"

Some drop the website name completetly:

"Your Account" or "Wish List" <-- on Amazon

I had a hard time finding those that listed their website first, but I found one:

"USPS.com - USPS Tracking"

My conclusion is that Jeshyr's suggestion of reversing the order of the page title [in http://www.buttonweavers.com/ui/forum.html#!threadId=211] is absolutely reasonable and one I'll undertake in this bug fix.

**Comment by dwvanstone** [comment]
I'd like to add another feature to the page title I notice in gmail & github. 

If I'm in my inbox in gmail and I get a message, the page title is as follows:

"Inbox (1) - [email adderss]"

Yahoo mail has as a page title: "(### unread) - [email] - Yahoo Mail"

I see this happen in github as well - if there's another message that comes in while you're working the page title changes to have a "(1)" inside indicating there's something to read. I'll explore this and see if it works for ButtonMen in the overview and the forum. (This will also help issue #1125.)

**Comment by yawetag** [comment]
Amazingly, the past semester of Digital Marketing is going to help here:

Companies, especially well-established ones, put their brand name last.
This is because Google ranks the first words higher than the last ones.
Also, if the title bar is shortened on the browser, the important
information has a better chance of being listed. Not to mention Google
Search, which may clip your title tag on the search results.

TL;DR: Best practice is to put the website name last.


Andrew Senger
asenger@gmail.com
